src/routes/about/index.svelte

```diff
@@ -1,8 +1,7 @@
 <script lang="ts">
-  import CommonHero from '$lib/components/sections/common-hero.svelte';
-  import OfficerProfileList from '$lib/components/about/officer-profile-list.svelte';
   import Spacing from '$lib/components/sections/spacing.svelte';
-  import Header from '$lib/components/about/what-is-acm.svelte';
+  import About from '$lib/components/about/about-us.svelte';
```

Comment:
Nit: Call this component `AboutUs` as to not confuse with the `About` page component.
```suggestion
  import AboutUs from '$lib/components/about/about-us.svelte';
```

Reply:
Fixed in commit 8a4f48a